packages/lesswrong/lib/collections/users/helpers.tsx

```diff
@@ -551,14 +551,25 @@ export const isMod = (user: UsersProfile|DbUser): boolean => {
 // TODO: I (JP) think this should be configurable in the function parameters
 /** Warning! Only returns *auth0*-provided auth0 Ids. If a user has an ID that
  * we get from auth0 but is ultimately from google this function will throw. */
-export const getAuth0Id = (user: DbUser) => {
+export const getAuth0IdIfUsernamePassword = (user: DbUser) => {
```

Comment:
This only allowed the "auth0" provider so I added a version which allows any provider. I have made a ticket to come back and combine them as I suspect the original restriction to "auth0" isn't necessary in most cases